moirae/models/ecm/components.py

```diff
@@ -131,11 +131,11 @@ def get_value(self,
         """
         Returns values of OCV at given SOC(s) and temperature(s).
         """
-        ocv = enforce_dimensions(self.ocv_ref.get_value(soc=soc), 0)
```

Comment:
Lovely that we can get rid of these calls to the enforce_dimensions functions.

Reply:
Agreed!